torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14802,6 +14802,8 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
             DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_variant_consistency_jit'),
             # Skip Nvfuser
             DecorateInfo(unittest.skip('Skipped!'), 'TestCudaFuserOpInfo'),
+            # RuntimeError: t == DeviceType::CUDA INTERNAL ASSERT FAILED
```

Comment:
Have the Jiterated ops fail this new test seems conceptually odd -- they should probably be skipping it instead, and without requiring an explicit skip. Maybe their `dtypes` needs to change to `dtypesIfCUDA` to signify they don't support CPU inputs better?

Reply:
This is decorated with :

```
decorators=[
            onlyCUDA,
```

Is there a way I can use this?
Another possibility is change the comment to just `# This is not implemented on CPU`  if `t == DeviceType::CUDA ` is not important.

Reply:
We could think about reusing the decorator, but I think the most natural way for an op to signal it only supports the CUDA device type is to define `dtypes` as empty and then define `dtypesIfCUDA`

Reply:
This test ended up being skipped because the numpy `ref=` is defined